app/models/case_information.rb

```diff
@@ -23,22 +25,67 @@ def latest_early_allocation
   validates :manual_entry, inclusion: { in: [true, false], allow_nil: false }
   validates :nomis_offender_id, presence: true, uniqueness: true
 
-  validates :local_divisional_unit, :team, presence: true, unless: ->{ manual_entry }
+  validates :team,
+            presence: { message: "You must select the prisoner's team" },
+            unless:
+            proc { |c|
+              c.manual_entry &&
+              (c.probation_service == 'Scotland' ||
+              c.probation_service == 'Northern Ireland')
+            }
 
-  validates :welsh_offender, inclusion: {
-    in: %w[Yes No],
-    allow_nil: false,
-    message: 'Select yes if the prisoner’s last known address was in Wales'
-  }
-  validates :tier, inclusion: { in: %w[A B C D], message: 'Select the prisoner’s tier' }
+  validates :tier, inclusion: { in: %w[A B C D N/A], message: "Select the prisoner's tier" }
 
   validates :case_allocation, inclusion: {
```

Comment:
I wonder if it might have been better to allow nil rather than the special value 'N/A'? (here in in the tier field)

Reply:
I think tier should still be mandatory unless the offender is Scotland/NI ?

Reply:
I tend to agree – it feels like `nil` might be a more meaningful value for 'tier' and 'case allocation' when they aren't set, and then display that `nil` value as a human-friendly `"N/A"` on the page.